src/Microsoft.DiaSymReader/SymUnmanagedFactory.cs

```diff
@@ -41,38 +53,51 @@ internal static string DiaSymReaderModuleName
 
         [DefaultDllImportSearchPaths(DllImportSearchPath.AssemblyDirectory | DllImportSearchPath.SafeDirectories)]
         [DllImport(DiaSymReaderModuleName32, EntryPoint = CreateSymReaderFactoryName)]
-        private static extern void CreateSymReader32(ref Guid id, [MarshalAs(UnmanagedType.IUnknown)] out object symReader);
+        private static unsafe extern void CreateSymReader32([MarshalAs(UnmanagedType.LPStruct)] Guid id, IntPtr* symReader);
```

Comment:
Changing `object` to `IntPtr*` seems like a breaking change.

Reply:
Yes, but it is a `private` method so not exposed.

Reply:
Ah, missed that. Thx.